nemo/collections/tts/modules/audio_codec_modules.py

```diff
@@ -677,6 +678,11 @@ def __init__(
             padding_mode=pad_mode,
         )
         self.conv = nn.utils.parametrizations.weight_norm(conv)
+        if activation is not None:
+            self.activation = CodecActivation(activation=activation, channels=out_channels)
+        else:
+            self.activation = None
```

Comment:
Can you refactor this to reduce the if statement during forward
```suggestion
        self.activation = torch.nn.Identity()
        if activation is not None:
            self.activation = CodecActivation(activation=activation, channels=out_channels)
```

Reply:
Alternatively, you could allow None to CodecActivation and make it return Identity, but that might cause silent errors.